modules/claims_api/spec/lib/claims_api/evidence_waiver_pdf_constructor/pdf_spec.rb

```diff
@@ -2,8 +2,13 @@
 
 require 'rails_helper'
 require 'claims_api/evidence_waiver_pdf/pdf'
-require_relative '../../support/pdf_matcher'
+require_relative '../../../support/pdf_matcher'
 
+# Helpful testing hints
+#
+# `cp #{generated_pdf} #{expected_pdf}`; sleep 1
+# `open #{generated_pdf} #{expected_pdf}`; sleep 1
```

Comment:
random, wouldn't the `cp` make them the same and the `open` be used to visually compare them but they'd be the same?

Reply:
These are testing tools, not generally used at the same time. If you make changes to your form, you can copy it to the expected. In another scenario, if your spec is failing, you can open the two pdfs up side by side to see what is different before blindly copying the generated to equal the expected.